server/mappers/csraReviewToSummaryListMapper.ts

```diff
@@ -55,7 +55,9 @@ export default (
     {
       key: { text: 'Next review date' },
       value: {
-        text: formatDate(new Date(csraAssessment.nextReviewDate).toISOString(), 'long'),
+        text: csraAssessment.nextReviewDate
+          ? formatDate(new Date(csraAssessment.nextReviewDate).toISOString(), 'long')
```

Comment:
`nextReviewDate` should be an ISO date already so no need to go string->Date->string